src/translator/service.h

```diff
@@ -99,9 +93,9 @@ class Service {
   /// some member in the Response, also specify any additional configurable
   /// parameters.
   std::future<Response> translate(std::string &&source,
-                                  ResponseOptions options);
+                                  ResponseOptions options = ResponseOptions());
```

Comment:
Did something change to require the explicit default constructor?

Reply:
I probably should have done this in a separate PR but it was too painful to see two separate functions:

`std::future<Response> translate(std::string &&source)` that just creates a default ResponseOptions

`std::future<Response> translate(std::string &&source, ResponseOptions opions)`

Reply:
I'm pretty sure i had complained about that somewhere...